src/components/todoitems/TodoItems.js

```diff
@@ -3,16 +3,18 @@ import './todoitems.css'
 import { FaTrash } from 'react-icons/fa';
 
 function TodoItems({ onCheck, onClick, data }) {
-
     return <div className='todoitem' >
-        <div>
+        <div className='div1'>
```

Comment:
className should be meaningfull
relace div1 with a meaningfull name

Reply:
done